packages/framework/client-logger/external-telemetry/api-report/external-telemetry.api.md

```diff
@@ -0,0 +1,73 @@
+## API Report File for "@fluidframework/external-telemetry"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ICriticalContainerError } from '@fluidframework/container-definitions';
+import type { IFluidContainer } from '@fluidframework/fluid-static';
+
+// @beta
+export interface ContainerConnectedTelemetry extends IContainerTelemetry {
+    // (undocumented)
```

Comment:
Before you merge, please ensure all of the `(undocumented)` members here have been documented. For many, you probably just want to use `{@inheritDoc Foo.bar}`.

Reply:
Updated in commit 20a366b, there should be no more undocumented pieces here.